### DISCUSSION TITLE
error building lotus

### DISCUSSION THREAD

**Discussion by tr4nscr1pt** [filecoin-project/lotus#4644] [Storage Provider]
CXXSTDLIB_x86_64-unknown-linux-gnu = None
  CXXSTDLIB_x86_64_unknown_linux_gnu = None
  HOST_CXXSTDLIB = None
  CXXSTDLIB = None
  cargo:rustc-link-lib=stdc++
  cargo:warning=couldn't execute `llvm-config --prefix` (error: No such file or directory (os error 2))
  cargo:warning=set the LLVM_CONFIG_PATH environment variable to the full path to a valid `llvm-config` executable (including the executable itself)

  --- stderr
  thread 'main' panicked at 'function not loaded: clang_createIndex', /home/user/.cargo/registry/src/github.com-1ecc6299db9ec823/clang-sys-0.29.3/src/lib.rs:1659:1
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: build failed
+ rm -f /tmp/tmp.uUcX133h9r
make[1]: *** [Makefile:11: .install-filcrypto] Error 101
make[1]: Leaving directory '/home/user/lotus/extern/filecoin-ffi'
make: *** [Makefile:37: build/.filecoin-install] Error 2

**Comment by jennijuju** [comment]
have you installed all the dependency in https://docs.filecoin.io/get-started/lotus/installation/#system-specific?

Whats your system version?

(also, converting to discussion)
